input.go

```diff
@@ -92,6 +92,29 @@ func IsMouseButtonPressed(mouseButton MouseButton) bool {
 	return uiDriver().Input().IsMouseButtonPressed(driver.MouseButton(mouseButton))
 }
 
+// GamepadGUID returns a string with the uuid.
```

Comment:
Question: How do we use GUID of the gamepads? This does not work on browsers anyway.

Reply:
I mostly implemented this because I wasn't sure what glfw methods would give me the information I needed. I don't actually utilize this anywhere.

However, its not crazy to think I might want to leverage this in the future for complex controller mapping.
ie. Users can setup certain profiles for certain controllers, the GUID could be used to differentiate different controller types.

Reply:
> However, its not crazy to think I might want to leverage this in the future for complex controller mapping.

Then, how about adding this when we think this is necessary?

Reply:
Sure, I'm happy to take this out of the PR. Want it omitted completely or just commented out? My thinking being the latter having less friction to bring back in later if someone wants or needs to.

Reply:
Hm actually, upon further investigation, it looks like this GUID value could be useful in helping me utilize an extensive / existing open-source gamepad mapping file:
https://github.com/gabomdq/SDL_GameControllerDB/blob/master/gamecontrollerdb.txt

Reply:
Well I understood GUID works, but you'd need to use `GamepadName` for potability in any cases. I prefer GUID in general but using names is inevitable so far.

Reply:
In my opinion, since GamepadGUID is more robust, ideally you want to use that *if* you have it and then fallback to using GamepadName if its unavailable / empty string.

Reply:
OK that makes sense. I hope Gamepad API will have a function for GUID in the future...